### PR TITLE
Dispatcher

### PR DESCRIPTION
**Pull request by CHIAHSINHOU**
dispatcher

resultUpdater need to be put under /var/www and change the the path where dispatcherFunction.php is.

### CODE REVIEW

File: VM/judge/runcode.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```
Comment (bruce3557) [inline VM/judge/runcode.sh]: Is this file same with another in in #1 ?

File: VM/judge/judge.sh [hunk 2]
```diff
@@ -0,0 +1,522 @@
+#!/bin/bash
```
Comment (bruce3557) [inline VM/judge/judge.sh]: Same as previous comment

File: dispatcher/dispatcher.php [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<?php
+/************************
+dispatcher.php
+This process continuously checks whether there exist unjudged submissions.
+If there's unjudged submissions, start 'submit_curl.php' process to send it to a available judge VM.
+**************************/
+
+session_start();
```
Comment (bbiiggppiigg) [inline dispatcher/dispatcher.php]: Is session needed in backend ?

File: dispatcher/resultUpdater.php [hunk 4]
```diff
@@ -0,0 +1,81 @@
+<?php
+	session_start();
```
Comment (bbiiggppiigg) [inline dispatcher/resultUpdater.php]: Are sessions need in backend?

**Comment by bruce3557** [comment]
+1

**Pull request merged**